tests/pages/firefox/new/download.py

```diff
@@ -22,7 +22,10 @@ def is_download_button_displayed(self):
         return self.is_element_displayed(*self._download_button_locator)
 
     def download_firefox(self):
-        self.find_element(*self._download_button_locator).click()
+        link = self.find_element(*self._download_button_locator)
+        href = link.get_attribute('href')
+        self.set_attribute(link, att_name='href', att_value=href + '?automation=true')
```

Comment:
It's not ideal having tests modify an in-page link, but for the purpose what we're doing here I think this is likely OK.

Reply:
Yeah. Agreed. Can't think of a better way to go about it though.